README.md

```diff
@@ -14,22 +15,22 @@ Visual studio [![Build status](https://ci.appveyor.com/api/projects/status/sm9jx
 folder structure
 --------
 
-This release of OF comes with several folders:
```

Comment:
intent of this sentence is, i believe, to explain what the folders mean. i feel repeating the subject ("folders") to clarify is sometimes good.